python/lsst/rubintv/production/guiders.py

```diff
@@ -191,10 +191,10 @@ def getRubinTvTableEntries(self, metrics: DataFrame) -> dict[str, str]:
         """
         rubinTVtableItems: dict[str, str] = {}
         for key, value in KEY_MAP.items():
-            if key in metrics:
+            try:
                 rubinTVtableItems[value] = f"{metrics[key].values[0]}"
-            else:
-                self.log.warning(f"Key {key} not found in metrics DataFrame columns")
+            except (KeyError, IndexError):
+                self.log.warning(f"Key {key} not found in metrics DataFrame columns or has not values")
```

Comment:
```suggestion
                self.log.warning(f"Key {key} not found in metrics DataFrame columns or has no values")
```
Unless you want to sound Shakespearean.